docs/user/security/audit-logging.asciidoc

```diff
@@ -128,38 +128,38 @@ Refer to the corresponding {es} logs for potential write errors.
 | `unknown` | User is updating a connector.
 | `failure` | User is not authorized to update a connector.
 
-.2+| `alert_update`
-| `unknown` | User is updating an alert.
-| `failure` | User is not authorized to update an alert.
+.2+| `rule_update`
+| `unknown` | User is updating a rule.
+| `failure` | User is not authorized to update a rule.
 
-.2+| `alert_update_api_key`
-| `unknown` | User is updating the API key of an alert.
-| `failure` | User is not authorized to update the API key of an alert.
+.2+| `rule_update_api_key`
+| `unknown` | User is updating the API key of a rule.
+| `failure` | User is not authorized to update the API key of a rule.
 
-.2+| `alert_enable`
-| `unknown` | User is enabling an alert.
-| `failure` | User is not authorized to enable an alert.
+.2+| `rule_enable`
+| `unknown` | User is enabling a rule.
+| `failure` | User is not authorized to enable a rule.
 
-.2+| `alert_disable`
-| `unknown` | User is disabling an alert.
-| `failure` | User is not authorized to disable an alert.
+.2+| `rule_disable`
+| `unknown` | User is disabling a rule.
+| `failure` | User is not authorized to disable a rule.
 
-.2+| `alert_mute`
+.2+| `rule_mute`
+| `unknown` | User is muting a rule.
+| `failure` | User is not authorized to mute a rule.
+
+.2+| `rule_unmute`
+| `unknown` | User is unmuting a rule.
+| `failure` | User is not authorized to unmute a rule.
+
+.2+| `rule_alert_mute`
 | `unknown` | User is muting an alert.
 | `failure` | User is not authorized to mute an alert.
 
-.2+| `alert_unmute`
+.2+| `rule_alert_unmute`
```

Comment:
Should this be `rule_unmute`?

Reply:
Same as above: This is meant to replace `alert_instance_unmute` because in our terminology update `alert` becomes `rule` and `alert instance` becomes `alert`.

Reply:
Ah, makes sense!